src/components/TaskList/TaskList.jsx

```diff
@@ -0,0 +1,27 @@
+import PropTypes from "prop-types";
+import { useSelector } from "react-redux";
+import TaskCard from "@components/TaskCard";
+import CreateTaskCard from "@components/CreateTaskCard";
+import "./TaskList.scss";
+
+function TaskList({ isTaskCreating, onTaskCreation }) {
+    const tasks = useSelector((state) => state.tasks.tasks);
```

Comment:
can't we use any other name here? `state.tasks.tasks` looking a bit confusing

Reply:
@ray-wtag bhaia, I've changed it from `state.tasks.tasks` to `state.todo.tasks`. Thanks!